app/views/analytics/districts/_analytics_table.html.erb

```diff
@@ -22,8 +22,8 @@
       <thead>
       <tr>
         <th></th>
-        <th class="row-label"><strong>Total</strong></th>
-        <th class="row-label" colspan="6"><strong>Patients with BP taken</strong></th>
+        <th class="row-label"><strong>Cumulative patients</strong></th>
+        <th class="row-label" colspan="6"><strong>Follow-ups</strong></th>
```

Comment:
Nice. Since this is the districts table, I think the simpler language is better 👍

Reply:
The "Follow ups" column was renamed to "Patients with BP taken", since theese are not follow ups at that facility anymore (these could be recorded at any facility). Just wanted to double check that this change is intended.

(For later) Will `Cumulative patients` be the ubiquitous term for "Total" patients? This number is being shown on Missed Visits and BP control as `Total patients`.

Reply:
This change is only for the district table, where both will be the same.

Reply:
Never mind, the facility-wise breakdown below will differ.